sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -130,36 +130,36 @@ The Text Analytics client library provides a [TextAnalyticsClient][text_analytic
 asynchronous operations to access a specific use of Text Analytics, such as language detection or key phrase extraction.
 
 ### Text input
-A **text input**, sometimes called a **document**, is a single unit of input to be analyzed by the predictive models
-in the Text Analytics service. Operations on Text Analytics client may take a single text input or a collection
-of inputs to be analyzed as a batch. 
+A **text input**, also called a **document**, is a single unit of document to be analyzed by the predictive models
+in the Text Analytics service. Operations on Text Analytics client may take a single document or a collection
+of documents to be analyzed as a batch. 
```

Comment:
Files under the  sample folder also need this update.
https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/AnalyzeSentiment.java#L27